exchanges/orderbook/depth_test.go

```diff
@@ -624,3 +625,836 @@ func TestIsValid(t *testing.T) {
 		t.Fatalf("received: '%v' but expected: '%v'", d.IsValid(), false)
 	}
 }
+
+func TestGetBaseFromNominalSlippage(t *testing.T) {
+	t.Parallel()
+	depth := NewDepth(id)
+	depth.LoadSnapshot(bid, ask, 0, time.Time{}, true)
+	_, err := depth.GetBaseFromNominalSlippage(10, 1355.5)
+	if !errors.Is(err, errNotEnoughLiquidity) {
+		t.Fatalf("received: '%v' but expected: '%v'", err, errNotEnoughLiquidity)
+	}
+
+	// First and second price
+	amt, err := depth.GetBaseFromNominalSlippage(0.018443378827001, 1336)
```

Comment:
My own personal note. I need to understand what's happening when I'm inputting a price of 3000

Reply:
FYI: The reference price in this case can be set e.g. TWAP, MID or BEST price etc. It was for a future consideration when we don't want to fall by a great degree from a certain point while deploying capital to a book.